### DISCUSSION TITLE
Transulent sidebars

### DISCUSSION THREAD

**Discussion by dvcrn** [notaapp/talk#309] [Feature Requests]
Obsidian recently dropped 0.16.1 which drastically improved how transparency is handled on macOS. Instead of having everything transparent, it now makes the sidebars (left + right) transulenct which makes the entire application flow nicer with the rest of the OS

<img width="455" alt="Screen Shot 2022-09-14 at 17 33 14" src="https://user-images.githubusercontent.com/688326/190103805-2732f6f0-ff14-434b-b4bf-6b78641b40a8.png">

Would love to see something similar in Nota!

**Comment by astoilkov** [comment]
Cool. Thanks. We will look into this.
